### ISSUE TITLE
Reputation score is wrong in some cases

### ISSUE THREAD

**Issue by espoem** [utopian-io/utopian.io#238]
#### Expected behavior

Reputation score for codingdefined should be 61.

![image](https://user-images.githubusercontent.com/16621517/34482294-578a50c4-efb7-11e7-8ac0-8090fae111e3.png)

![image](https://user-images.githubusercontent.com/16621517/34482295-5afb3f7a-efb7-11e7-9bc3-a85e4d8fc564.png)

#### Actual behavior

The reputation score is 70 in Utopian.

![image](https://user-images.githubusercontent.com/16621517/34482302-66eca0f8-efb7-11e7-9630-52eda4680fd7.png)

#### How to reproduce

1. Open https://utopian.io/@codingdefined
2. Compare to https://steemd.com/@codingdefined or https://steemit.com/@codingdefined

* Browser: Chrome
* Operating system: Windows 10

**Comment by liraz** [comment]
Cannot reproduce.
It shows 61, not 70... maybe it was a temporary sync issue.

**Comment by codingdefined** [comment]
@liraz Yes it was showing 70 when my reputation just touched 61, and after I got upvote it is not showing now. We can actually find out if anyone else also getting it or not who is suppose to reach reputation 61

**Comment by espoem** [comment]
I can provide screenshots from other apps. @liraz Perhaps a temprary sync issue as you say.

Steemnow
![image](https://user-images.githubusercontent.com/16621517/34557388-5b83dcca-f13a-11e7-82e8-4606e9b58fd9.png)

Busy
![image](https://user-images.githubusercontent.com/16621517/34557392-624aff3e-f13a-11e7-8e5c-11a232e14965.png)
